Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -496,7 +496,6 @@ class BrowserViewController: UIViewController {
         }
 
         updateTabCountUsingTabManager(tabManager, animated: false)
-        clipboardBarDisplayHandler?.checkIfShouldDisplayBar()
```

Comment:
So just for my understanding, this was creating an extra check that isn't necessary since we have checks embedded in ClipboardBarDisplayHandler already?